contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -770,90 +775,6 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
-     *      In particular, we want to be sure that contracts cannot trigger an INVALID_STATE_ACCESS
-     *      flag and then revert to reset the flag. We're able to do this by making an external
-     *      call from `ovmCREATE` and `ovmCREATE2` to `safeCREATE`, which can capture and relay
-     *      information before reverting.
-     * @param _address Address of the contract to associate with the one being created.
-     * @param _bytecode Code to be used to create the new contract.
-     */
-    function safeCREATE(
-        address _address,
-        bytes memory _bytecode
-    )
-        override
-        public
-    {
-        // Since this function is public, anyone can attempt to directly call it. We need to make
-        // sure that the OVM_ExecutionManager itself is the only party that can actually try to
-        // call this function.
-        if (msg.sender != address(this)) {
-            return;
-        }
-
-        // We need to be sure that the user isn't trying to use a contract creation to overwrite
-        // some existing contract. On L1, users will prove that no contract exists at the address
-        // and the OVM_FraudVerifier will populate the code hash of this address with a special
-        // value that represents "known to be an empty account."
-        if (_hasEmptyAccount(_address) == false) {
-            _revertWithFlag(RevertFlag.CREATE_COLLISION);
-        }
-
-        // Check the creation bytecode against the OVM_SafetyChecker.
-        if (ovmSafetyChecker.isBytecodeSafe(_bytecode) == false) {
-            _revertWithFlag(RevertFlag.UNSAFE_BYTECODE);
-        }
-
-        // We always need to initialize the contract with the default account values.
-        _initPendingAccount(_address);
-
-        // Actually deploy the contract and retrieve its address. This step is hiding a lot of
-        // complexity because we need to ensure that contract creation *never* reverts by itself.
-        // We cover this partially by storing a revert flag and returning (instead of reverting)
-        // when we know that we're inside a contract's creation code.
-        address ethAddress = Lib_EthUtils.createContract(_bytecode);
-
-        // Contract creation returns the zero address when it fails, which should only be possible
-        // if the user intentionally runs out of gas. However, we might still have a bit of gas
-        // left over since contract calls can only be passed 63/64ths of total gas,  so we need to
-        // explicitly handle this case here.
-        if (ethAddress == address(0)) {
-            _revertWithFlag(RevertFlag.CREATE_EXCEPTION);
-        }
-
-        // Here we pull out the revert flag that would've been set during creation code. Now that
-        // we're out of creation code again, we can just revert normally while passing the flag
-        // through the revert data.
-        if (messageRecord.revertFlag != RevertFlag.DID_NOT_REVERT) {
```

Comment:
This flag is no longer used, anywhere in this file. 

It should probably _at least_ be reset in the `_resetContext()`. But maybe there are also implications for that in the new `_handleExternalCreate()`

Reply:
Yeah we should be able to remove this flag now.

Reply:
I'm not sure of that. We at least need a flag corresponding to the default context prior to `initializeContext` and after `resetContext`.

Reply:
With this change, the value luckily does is not stored anymore (see change to `iOVM_EXecutionManager.messageRecord`), so I think it can genuinely be removed now.